src/Sylius/Behat/Context/Setup/TaxContext.php

```diff
@@ -71,31 +72,28 @@ public function __construct(
      * @Transform /^"([^"]+)" tax category$/
      * @Transform /^tax category "([^"]+)"$/
      */
-    public function castTaxCategoryNameToTaxCategory($taxCategoryName)
+    public function getTaxCategoryByName($taxCategoryName)
     {
         $taxCategory = $this->taxCategoryRepository->findOneBy(['name' => $taxCategoryName]);
         if (null === $taxCategory) {
-            throw new \Exception('Tax category with name "'.$taxCategoryName.'" does not exist');
+            throw new \InvalidArgumentException('Tax category with name "'.$taxCategoryName.'" does not exist');
         }
 
         return $taxCategory;
     }
 
     /**
-     * @Given /^store has "([^"]+)" tax rate of ([^"]+)% for "([^"]+)" within ("([^"]+)" zone)$/
+     * @Given store has :taxRateName tax rate of :taxRateAmount% for :taxCategoryName within :zone zone
+     * @Given /^store has "([^"]+)" tax rate of ([^"]+)% for "([^"]+)" for (the rest of the world)$/
```

Comment:
Changed my mind, it is necessary: to handle ``(the rest of the world)`` with transformer, step definition must be regex ;/